.github/workflows/continous-integration.yml

```diff
@@ -0,0 +1,18 @@
+name: Continous Integration
+on: [push, pull_request]
+jobs:
+  run-tests:
+    name: Run PHPUnit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+      
+      - name: Copy sample.env to .env
+        run: cp sample.env .env
+
+      - name: Build and start the containers
+        run: make up
+      
+      - name: Run the tests inside the container
+        run: docker exec tarallo_app_1 php /var/www/html/vendor/phpunit/phpunit/phpunit --configuration /var/www/html/phpunit.xml --teamcity
```

Comment:
You can simplify this command a bit, since you've set WORKDIR to /var/www/html. Also, you can use the generic container name "app". Please test this before changing it though.
```suggestion
        run: docker exec app php vendor/phpunit/phpunit/phpunit --configuration phpunit.xml --teamcity
```

Reply:
Should be fixed with a2bb80a